tests/test_configuration_manager.py

```diff
@@ -187,7 +187,7 @@ def create_conf_manager(tmpdir, cli_context: CliContext = None) -> Configuration
         app_name=APP_NAME,
         docker_image="invalid-image-name",
         seed_app_configuration_file=Path(BASE_DIR, "resources/test_app.yml"),
-        seed_templates_dir=Path(BASE_DIR, "resources/templates"),
```

Comment:
I suggest we should also be testing the new functionality around `configurable_templates`
Update the tests to check the files get dumped at least on an init

Reply:
Added simple test case